### PR TITLE
ARTESCA-11338: Fix the issue of Veeam welcome modal

### PR DESCRIPTION
**Pull request by ChengYanJin**
We change the condition to display the Veeam Welcome Modal from No Veeam Account to No Account at all.

### CODE REVIEW

File: src/react/ui-elements/Veeam/VeeamWelcomeModal.tsx [hunk 1]
```diff
@@ -25,37 +24,51 @@ const CustomModal = styled(Modal)`
...
    3. Storage Manager is logged in
    4. Have never click on the skip button
+   5. Not already in the Veeam configuration view
    */
   const isVeeamWelcomeModalEnabled =
     features.includes(VEEAM_FEATURE) &&
     isStorageManager &&
-    !isVeemAccountCreated &&
-    !isVeeamAssistantSkipped;
+    isZeroAccountCreated &&
```
Comment (JBWatenbergScality) [inline src/react/ui-elements/Veeam/VeeamWelcomeModal.tsx:46]: I feel like when we are in " loading status" for the account there is a possibility to have the modal displayed, and then once the loading is finished the modal is dismounted

Comment (ChengYanJin) [reply src/react/ui-elements/Veeam/VeeamWelcomeModal.tsx:46]: I believe the isZeroAccountCreated is false while loading the account? 

 ```js
 const isZeroAccountCreated = status === 'success' && accounts.length === 0;
 ```

**Review by JBWatenbergScality — APPROVED** [review]
LGTM !

**Pull request merged**